NotificationsMessageComposer.py

```diff
@@ -0,0 +1,9 @@
+''' Mimics composer. Composes and enqueues the message onto the pipeline.'''
```

Comment:
Done. Clean Code helps a lot with this. PEP-8 is nice but not exhaustive. Thanks for the recommendation!